app/models/skill_set.rb

```diff
@@ -1,2 +1,12 @@
 class SkillSet < ActiveRecord::Base
+  def attributes(display_price = 1)
+    {
+     "strength" => strength,
+    "dexterity" => dexterity,
+ "intelligence" => intelligence,
+        "speed" => speed,
+        "money" => display_price * money,
+       "health" => health
+     }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.